azkaban-hadoop-security-plugin/src/main/java/azkaban/security/HadoopSecurityManager_H_2_0.java

```diff
@@ -406,7 +403,7 @@ private void getToken(final String userToProxy) throws InterruptedException,
   private void cancelHiveToken(final Token<? extends TokenIdentifier> t)
       throws HadoopSecurityManagerException {
     try {
-      final IMetaStoreClient hiveClient = this.hiveMetaStoreClientFactory.create();
+      final IMetaStoreClient hiveClient = new IMetaStoreClientFactory(new HiveConf()).create();
```

Comment:
Valid point. We currently don't have a mapping from token to a conf or a client. Thus, I think the cancellation of "other" delegation tokens should be broken. I created #1261 for this.